### PR TITLE
feat: Add km_KH locale to component

### PR DESCRIPTION
**Pull request by vireakkeosokvibol**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [x] Other (Localization)

### 💡 Background and solution
Khmer Locales were missing

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |     Added locales for Khmer translation.      |
| 🇨🇳 Chinese |    新增高棉语语言包。       |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Comment by afc163** [comment]
Could you add test case?

**Comment by vireakkeosokvibol** [comment]
> Could you add test case?

Do really apologize. Can anybody has experience with test unit help contribute the test? I'm really appreciated to the contribution.

**Comment by yoyo837** [comment]
FYI https://github.com/ant-design/ant-design/pull/32106#issuecomment-915964752

**Comment by yoyo837** [comment]
Some time later, a detailed docs about add a new locale will be added.

**Comment by vireakkeosokvibol** [comment]
> Some time later, a detailed docs about add a new locale will be added.

I'm done with the test unit and docs. But for about **i18n.zh-CH.md** i'm not really sure what Khmer mean in Chinese assuming its **高棉语**. Please have a correction if its wrong.

**Comment by yoyo837** [comment]
> > FYI #32106 (comment)
> 
> I'm done with the test unit and docs. But for about **i18n.zh-CH.md** i'm not really sure what Khmer mean in Chinese assuming its **高棉语**. Please have a correction if its wrong.

It's right.

**Comment by afc163** [comment]
feature branch please.

**Comment by vireakkeosokvibol** [comment]
Already switch branch to feature

> feature branch please.

File: components/input/style/mixin.less [hunk 1]
```diff
@@ -306,6 +301,12 @@
         border-bottom-left-radius: @border-radius-base;
       }
     }
+
+    &:not(:first-child),
+    .@{ant-prefix}-input-search &:not(:first-child) {
+      border-top-left-radius: 0;
+      border-bottom-left-radius: 0;
+    }
```
Comment (afc163) [inline components/input/style/mixin.less:309]: Mixed other commits.

**Comment by afc163** [comment]
图片

This commit is not belong to your changes.

**Comment by vireakkeosokvibol** [comment]
@afc163 can you suggest a revision to pass all the CI pipeline ?

**Comment by yoyo837** [comment]
Rebase origin/feature and push by force.

**Comment by vireakkeosokvibol** [comment]
> Rebase origin/feature and push by force.

Done rebase. Thank for your suggestion.

**Comment by yoyo837** [comment]
Looks like you still need:
1. Rebase origin/feature and push by force. (You already did.)
2. `rm -rf node_modules`
3. `npm install`
4. `npm test -- -u`
5. Push all file changes.

**Review by afc163 — CHANGES_REQUESTED** [review]
图片

This commit is not belong to your changes.

**Comment by vireakkeosokvibol** [comment]
> 图片
> 
> This commit is not belong to your changes.

Have dropped that commit.

**Review by afc163 — APPROVED** [review]

**Pull request merged**